app/boards/arm/preonic/preonic_rev3.dts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <st/f3/stm32f303Xc.dtsi>
```

Comment:
```suggestion
#include <st/f3/stm32f303Xc.dtsi>
#include <st/f3/stm32f303c(b-c)tx-pinctrl.dtsi>
```

An extra include is needed to pull in the pinctrl nodes.

Reply:
done